### PR TITLE
API tweaks and fixes

### PR DESCRIPTION
**Pull request by ouziel-slama**
- [x] Fix earned and burned fields
- [x] Issuer=None when no issuer
- [x] Add BTC sent in DISPENSE event
- [x] Add `event_name` filter for all `*/events` routes
- [x] Route to get assets by address
- [x] Route to get transactions by address
- [x] Routes to get dividend by address
- [x] Add action filter for `get_credits*` and `get_debits*`
- [x] Routes to get orders by address
- [x] Remove balances==0 in Get Address Balances route
- [x] Include Number of Counterparty Transactions in /v2/blocks/<block> 
- [x] Add block_time in every result containing block_index
- [x] Document when `params` could be different for the same event
- [x] Document possible values for all parameters
- [x] Update the project [release notes](release-notes/)
- [x] Update `apiary.apib` and `Documentation` repo 

Comes with https://github.com/CounterpartyXCP/Documentation/pull/201/files

### CODE REVIEW

**Comment by ouziel-slama** [comment]
Here updated doc from this PR: https://ouzielslama.docs.apiary.io/

File: release-notes/release-notes-v10.2.0.md [hunk 1]
```diff
@@ -54,9 +66,14 @@ Note: A reparse from block 819250 is automatically launched during the update.
...
 * Database connection pooling.
+* If `verbose=true`, enrich results containing `block_index` with `block_time`.
+* Added an `action` filter for the `*/credits` and `*/debits` routes.
+* Add `issuer=None` in XCP and BTC asset information.
+* Excludes zero balances in the results of `/v2/addresses/<address>/balances`.
+* Add BTC sent in `DISPENSE` event.
 
 ## Command-Line Interface
-
+* `-v` for `DEBUG` level, `-vv` for `EVENT` level, `-vvv` for `TRACE` level
```
Comment (adamkrellenstein) [inline release-notes/release-notes-v10.2.0.md:76]: is there a way to do this with `--verbose`?

**Review by adamkrellenstein — APPROVED** [review]
Let's make sure that 100% of the changes are represented in the release notes. I didn't go through them one-by-one but I think at least the `event_name` filter is missing.

File: release-notes/release-notes-v10.2.0.md (hunk 1 above)
Comment (adamkrellenstein) [reply release-notes/release-notes-v10.2.0.md]: great. I didn't see this in the `--help` text or the docs, but maybe I missed it. would you mind checking?

**Pull request merged**